The job **`pr-check-2`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/16987541763/job/48159668627) [(plain enhanced)](https://triage.rust-lang.org/gha-logs/rust-lang/rust/48159668627) [(plain)](https://github.com/rust-lang/rust/commit/d33e1967a6a75da87a9b73d35382e9091ea7bbca/checks/48159668627/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
    Checking miri v0.1.0 (/checkout/src/tools/miri)
error[E0609]: no field `export_name` on type `&CodegenFnAttrs`
   --> src/tools/miri/src/shims/foreign_items.rs:149:32
    |
149 |                     if !(attrs.export_name.is_some()
    |                                ^^^^^^^^^^^ unknown field
    |
    = note: available fields are: `flags`, `inline`, `optimize`, `symbol_name`, `link_ordinal` ... and 10 others

For more information about this error, try `rustc --explain E0609`.
```

</details>

--- a/compiler/rustc_builtin_macros/src/lib.rs
+++ b/compiler/rustc_builtin_macros/src/lib.rs
@@ -8,7 +8,6 @@
 #![doc(html_root_url = "https://doc.rust-lang.org/nightly/nightly-rustc/")]
 #![doc(rust_logo)]
 #![feature(assert_matches)]
-#![feature(autodiff)]
 #![feature(box_patterns)]
 #![feature(decl_macro)]
 #![feature(if_let_guard)]

